api/src/main/java/com/github/retrooper/packetevents/wrapper/play/client/WrapperPlayClientSetBeaconEffect.java

```diff
@@ -69,4 +80,18 @@ public int getSecondaryEffect() {
     public void setSecondaryEffect(int secondaryEffect) {
         this.secondaryEffect = secondaryEffect;
     }
-}
+
+    private int readEffect() {
```

Comment:
I'd suggest readOptionalEffect() to avoid future conflicts if readEffect() was added to PacketWrapper